File: spec/code_samples/Perl_perl5/boards@notes/patch.pl
#!/usr/bin/perl
use Modern::Perl;
use Data::Dumper;
use HTTP::Request::JSON;
use LWP::UserAgent::JSON;
my $request = HTTP::Request::JSON->new("PATCH",
"https://www.directfreight.com/api/boards/notes");
$request->content('{"body":"<ADD STRING VALUE>","title":"<ADD STRING VALUE>"}');
my $browser = LWP::UserAgent::JSON->new();
my $response = $browser->request($request);
print Dumper($response->json_content);

File: spec/code_samples/Php_http1/boards@notes/patch.php
<?php

HttpRequest::methodRegister('PATCH');
$request = new HttpRequest();
$request->setUrl('https://www.directfreight.com/api/boards/notes');
$request->setMethod(HttpRequest::HTTP_METH_PATCH);

$request->setBody('{"body":"<ADD STRING VALUE>","title":"<ADD STRING VALUE>"}');

try {
  $response = $request->send();

  echo $response->getBody();
} catch (HttpException $ex) {
  echo $ex;
}